src/NuGetPackageVerifier/Program.cs

```diff
@@ -25,6 +25,13 @@ public static int Main(string[] args)
 
             // TODO: Get this from the command line
             var ignoreAssistanceMode = IgnoreAssistanceMode.ShowAll;
+            var hideInfoLogs = false;
+
+            if (args.Length > 0 && string.Equals("--quiet", args[0], StringComparison.OrdinalIgnoreCase))
```

Comment:
Sure. The `N` at the start of the line was for normal, the `I` was for Info. So we would have a couple of visible text.